omnibus/package-scripts/agent-dmg/postinst

```diff
@@ -54,6 +53,13 @@ mkdir -vp "$RUN_DIR"
 chown -vR "${INSTALL_USER}:admin" "$RUN_DIR"
 chmod -v 755 "$RUN_DIR"
 
+# Create IPC directory for WiFi data exchange between GUI and agent
+WIFI_IPC_DIR=/var/run/datadog-agent
+echo "# Setting up WiFi IPC directory"
+mkdir -vp "$WIFI_IPC_DIR"
+chown -v root:wheel "$WIFI_IPC_DIR"
+chmod -v 1777 "$WIFI_IPC_DIR"  # Sticky bit: users can create but not delete others' files
```

Comment:
While the sticky bit prevents deletion, this could enable:
- socket name squatting (creating a socket before the legitimate GUI starts)
- DoS attacks (filling the directory with sockets)
- confusion attacks if multiple users are logged in

... but I guess I'm overly pessimistic?